app/src/main/java/org/merakilearn/MainActivity.kt

```diff
@@ -7,22 +7,25 @@ import android.os.Bundle
 import android.os.Parcelable
 import android.view.View
 import android.widget.SearchView
-import android.widget.TextView
+import androidx.annotation.AttrRes
 import androidx.appcompat.app.AppCompatActivity
+import androidx.appcompat.widget.Toolbar
 import androidx.navigation.fragment.NavHostFragment
 import androidx.navigation.ui.setupWithNavController
 import kotlinx.android.parcel.Parcelize
 import kotlinx.android.synthetic.main.activity_main.*
 import org.koin.android.ext.android.inject
 import org.merakilearn.core.appopen.AppOpenDelegate
 import org.merakilearn.util.AppUtils
+import org.navgurukul.commonui.platform.ToolbarConfigurable
+import org.navgurukul.commonui.themes.getThemedColor
 
 @Parcelize
 data class MainActivityArgs(
     val clearNotification: Boolean
 ) : Parcelable
 
-class MainActivity : AppCompatActivity() {
+class MainActivity() : AppCompatActivity(), ToolbarConfigurable {
```

Comment:
Is there any need for empty constructor ?

Reply:
no i guess this was added by mistake, I will fix it